src/js/components/pages/tag-editor.jsx

```diff
@@ -151,22 +151,22 @@ var TagEditor = React.createClass({
     },
 
     getSceneTags: function (scene) {
-        
+
```

Comment:
We must have a different setting for whitespace, end of line or new line?

Doesn't cause me any issues

Reply:
I think i'm using newline, might be an editor setting